builder/main.lisp

```diff
@@ -0,0 +1,81 @@
+;;; Core
+
+(defun main ()
```

Comment:
It is more a common pattern to define the main function in the end of the file. Make it the code reading more linear